StRoot/StRefMultCorr/BadRun.h

```diff
@@ -66,16 +69,47 @@ const Int_t badrun_grefmult_2016[nBadRun_grefmult_2016] = {
   // Au+Au 200 GeV
-  20191005, 20191015, 20192001, 20193001, 20193019
+  20191005, 20191015, 20192001, 20193001, 20193019,
+  // Au+Au 4.59 AGeV (3.2 GeV)
+  20180005, 20180006, 20180019, 20180025, 20181016, 20182034, 20183001, 20183013, 20183014, 20183019,
+  // Au+Au 7.3 AGeV (1) = 3.9 GeV
+  20169051
 };
 
+//
+// Run 20
+// 1. Au+Au 31.2 AGeV (3) = 7.7 GeV
+// 2. Au+Au 7.3 AGeV (4) = 3.9 GeV
+// 3. Au+Au 5.75 AGeV (8) = 3.5 GeV
+//
+const Int_t nBadRun_refmult_2020 = 15;
+const Int_t badrun_refmult_2020[nBadRun_refmult_2020] = {
+  // Au+Au 31.2 AGeV (7.7 GeV)
+  21029002, 21029013, 21029027,
+  // Au+Au 7.3 AGeV (3.9 GeV)
+  21035008, 21035011, 21035012, 21035014,
+  // Au+Au 5.75 AGeV (3.5 GeV)
+  355020, 20355021, 21044023, 21045024, 21045025, 21044027, 21044035, 21045004
```

Comment:
Typo: 355020 -> 20355020
```suggestion
  20355020, 20355021, 21044023, 21045024, 21045025, 21044027, 21044035, 21045004
```